.storybook/mock-content/algoliaProduct.js

```diff
@@ -1,99 +1,106 @@
 import testImageURL from "../../resources/camera.jpg";
 
-// eslint-disable-next-line import/prefer-default-export
-export const algoliaProductMock = [
+const PRICING_ARRAY_ONLY_LIST = [
 	{
-		name: "Item 1",
-		image: testImageURL,
-		sku: "sku-1",
+		id: null,
+		currencyCode: "USD",
+		currencyDisplayFormat: "symbol",
+		currencyLocale: "en-US",
 		prices: [
 			{
-				amount: 26,
+				id: 70076,
```

Comment:
No need to add `id` if we are not using it

Reply:
updated -- thanks